mlflow/pyfunc/__init__.py

```diff
@@ -1,16 +1,33 @@
 # -*- coding: utf-8 -*-
 
 """
-The ``mlflow.pyfunc`` module defines a generic :ref:`filesystem format <pyfunc-filesystem-format>`
-for Python models and provides utilities for saving to and loading from this format. The format is
-self contained in the sense that it includes all necessary information for anyone to load it and
-use it. Dependencies are either stored directly with the model or referenced via a Conda
-environment.
+The ``python_function`` model flavor serves as a default model interface for MLflow python models.
+Any MLflow python model is expected to be loadable as a ``python_function`` model.
```

Comment:
Nit: Capitalize Python

```suggestion
The ``python_function`` model flavor serves as a default model interface for MLflow Python models.
Any MLflow Python model is expected to be loadable as a ``python_function`` model.
```